ckcp/openshift_dev_setup.sh

```diff
@@ -23,6 +23,7 @@ CR_TOSYNC=(
             runs.tekton.dev
             tasks.tekton.dev
             repositories.pipelinesascode.tekton.dev
+            networkpolicies.networking.k8s.io
```

Comment:
You need to add it to `kcp-registrar/register.sh` as well.

Reply:
added comment in the end as per our discussion during the standup.